CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Changelog
 
+## 1.25.2 2017-
+
+* Restored the jQuery dependency for the min.js version (liayn)
```

Comment:
Please change this to

`Restored the jQuery dependency for the min.js version. (liayn)`